tests/snapshots/test_multi_trace.json

```diff
@@ -7,6 +7,7 @@
     "span_id": 1,
```

Comment:
Reviewers: Make sure to scroll to the bottom, there are test changes after this large chunk of snapshot file updates.